microsoft-edge/webview2/concepts/data-privacy.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Data and privacy in WebView2
+description: WebView2 data collection and practices, privacy, and the 'Diagnostic data' setting.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2023

```

Comment:
```suggestion
ms.date: 05/16/2023
```
motivation: see if doc build reports a false 404

Reply:
this thread can be closed, per latest https://github.com/MicrosoftDocs/edge-developer/pull/2575/files